src/helpers/utilities.ts

```diff
@@ -399,3 +399,17 @@ export const fromWei = (number: BigNumberish): string =>
 export const delay = (ms: number): Promise<void> => {
   return new Promise(resolve => setTimeout(resolve, ms));
 };
+
+/**
+ * @desc Array.prototype.some repurposed for async iteration
```

Comment:
I wish to understand some day why we're adding "docs" in this file 😅

Reply:
Just following the file's pattern! They do show up in the VSCode code tools when you hover over this function which is nice 🤓

Reply:
why is michal such a hater?!?!